spec/support/features/in_person_helper.rb

```diff
@@ -34,39 +34,39 @@ def fill_out_address_form_ok
     choose t('in_person_proofing.form.address.same_address_choice_yes')
   end
 
-  def begin_in_person_proofing(_user = user_with_2fa)
+  def begin_in_person_proofing(_user = nil)
```

Comment:
The argument was previously unused, and the `user_with_2fa` method is non-trivial, so should help performance slightly to avoid calling it.

Reply:
Should we just remove the `_user` parameter for each of these methods where it's unused?

Reply:
> Should we just remove the `_user` parameter for each of these methods where it's unused?

I'm of two minds about it. I sorta wish we didn't ever need to pass the argument, but there are some instances where we do need to reference it ([example](https://github.com/18F/identity-idp/blob/e59299ec28510e8c60f672e3b0d778a9748772cb/spec/support/features/idv_step_helper.rb#L25)), and I'd rather we be consistent about the signature with these `complete_x_step` helpers than to (a) leave a developer unsure if they need to pass it and (b) make it a hassle if ever the helper did need to reference the user (i.e. updating all call sites). So the thinking with this is to expect it to be passed if it's known, acknowledging that it's not actually used currently, but leaving it open so that it could be used, and having a consistent argument signature.